src/api/ledger/balances.js

```diff
@@ -31,14 +34,26 @@ function getTrustlinesAsync(account, options, callback) {
     .catch(callback);
 }
 
+function getLedgerVersion(remote: Remote, optionValue?: number,
```

Comment:
Confusing name because `getLedgerVersion` already exists in the API, maybe something like `getLedgerVersionHelper`?